lib/situation_room/application.ex

```diff
@@ -0,0 +1,16 @@
+defmodule SituationRoom.Application do
+  use Application
+
+  @impl true
+  def start(_type, _args) do
+    # List all child processes to be supervised
+    children = [
+      SituationRoom.Repo,
+      {Plug.Cowboy, scheme: :http, plug: SituationRoom.Router, options: [port: 4001]}
```

Comment:
Port shouldn't be hard-coded.  This should come from an app config setting that is read from the ENV and then can default to 4001 if we want.